circ_blocks/examples/zxc.rs

```diff
@@ -23,15 +24,16 @@ use circ::cfg::{
     CircOpt,
 };
 use core::cmp::Ordering;
-use std::path::{Path, PathBuf};
+use std::path::Path;
+use std::path::PathBuf;
 
 use libspartan::{
-    instance::Instance, Assignment, InputsAssignment, MemsAssignment, SNARKGens, VarsAssignment,
-    SNARK,
+    instance::Instance, Assignment, InputsAssignment, MemsAssignment, VarsAssignment, SNARK,
 };
 use merlin::Transcript;
 use serde::{Deserialize, Serialize};
 use std::time::*;
+use std::time::*;
```

Comment:
Looks like we should do quite a bit of clean up?  This is just a duplicate of the line above.

Reply:
Yes you're right. This is from the merge conflict resolution. While I deleted a bunch of them, some might still slip through. I'm cleaning these up

Reply:
Have a look at https://github.com/scroll-tech/ceno-recursion/pull/46 for some automated cleanup I did.  I think that fixes most of these and makes the diff a bit smaller.

Reply:
Basically what I did for https://github.com/scroll-tech/ceno-recursion/pull/46 was:

Apply Ceno's formatting rules against `main` and make a separate PR for that.

Apply Ceno's formatting rules against the changes in your Goldilocks PR here, and make a PR for that.